Lab3/ComputerStore.WebUI/ApiControllers/AuthorizationApiController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using ComputerStore.DataAccessLayer.Models.Identity;
+using ComputerStore.WebUI.AppConfiguration;
+using ComputerStore.WebUI.Models;
+using ComputerStore.WebUI.Models.JwtToken;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace ComputerStore.WebUI.ApiControllers
+{
+    [Route("api/authorization")]
```

Comment:
It would be nice to have version component in the route.
Read more about AddVersioning.